### DISCUSSION TITLE
Retrieve Logs from Distant Hatchery (OpenStack)

### DISCUSSION THREAD

**Discussion by jqueguiner** [ovh/cds#6179] [Q&A] [answered]
When setting up Open Stack Hatchery logs are not displayed anymore in the UI nor in the CLI stream

I don't have a Graylog setup could it be that?
Is the hatchery sending the logs through the main API ?

thanks

**Comment by yesnault** [comment, marked as answer]
Hi, if the question is about hatchery logs: it's the same behaviour for all hatcheries. You should have log of the hatchery when you run the engine binary. You don't need to setup graylog, and it's not send on the api.
If the question if about the Jobs logs: your worker (openstack vm) have to reach the cdn service on tcp port. Is it the case? See section `[cdn.tcp]` in cdn service configuration.
